awscli/customizations/configure/exportcreds.py

```diff
@@ -69,6 +69,10 @@ def display_credentials(self, credentials):
         )
         if credentials.token is not None:
             output += f'export AWS_SESSION_TOKEN={credentials.token}\n'
+        if credentials.expiry_time is not None:
+            output += (
+                f'export AWS_CREDENTIAL_EXPIRATION={credentials.expiry_time}\n'
```

Comment:
I called this `AWS_CREDENTIALS_EXPIRATION` with an S in aws-export-credentials, could we make it the same here? https://github.com/benkehoe/aws-export-credentials/blob/0c53fa6afd4123d7a28550919cea85163a067605/aws_export_credentials/aws_export_credentials.py#L228

Reply:
I was basing it off of what's already supported in botocore (https://github.com/boto/botocore/pull/1187).  Looks like it's also supported in [javascript](https://github.com/aws/aws-sdk-js-v3/blob/e0697ed82d7eaa98ccc48337b516bf7f3b369e3a/packages/credential-provider-env/src/fromEnv.ts).

Reply:
Oh dang, I missed that when I was implementing it! I'll change aws-export-credentials to conform to it, then.

Reply:
Might be worth documenting in here as well: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html

Reply:
Is there a commensurate `aws_credential_expiration` value allowed in the profile config? If not, should there be?

Reply:
@benkehoe Thanks for pointing out that the environment variable is not documented there. We will follow up on this to get it documented.

For the a profile config version of `AWS_CREDENTIAL_EXPIRATION`, the CLI/botocore does not have one. I don't think the other SDKs have one either. It could potentially make sense. We have had internal discussions about it before I believe and I think the biggest concern is around potentially reloading the shared config file and handling any edge cases in reloading the shared config file.

If this is something that you'd like to see, I'd say recommend opening a feature request in the shared SDK GitHub repository to get a discussion going around it: https://github.com/aws/aws-sdk